automation/build-artifacts.sh

```diff
@@ -1,8 +1,28 @@
-#!/bin/bash -xe
+#!/bin/bash -e
 
 # Silence wine 'fixme' warnings
 export WINEDEBUG=fixme-all
 
+# Define VARS for make
+# Due to https://github.com/wixtoolset/issues/issues/5314
+# It is that the path will be under 260 chars
+# If we get an light.exe error "The system cannot find the file..."
+# This is probably the cause
+VIRTIO_WIN_DRIVERS_PATH=${1:-"$PWD/vwi"}
+
+if [ ! -e "${VIRTIO_WIN_DRIVERS_PATH}" -o \
+     ! -e "${VIRTIO_WIN_DRIVERS_PATH}/viorng" ] ; then
+    echo "Must pass in a directory containing virtio-win iso content. "
+    echo "If using virtio-win-pkg-scripts, this is something like "
+    echo "tmp*/make-driver-dir-output-*"
```

Comment:
We want to project to be the same for U/S and D/S so I don't like referencing "virtio-win-pkg-scripts", I suggest changing the error massage

Reply:
Makes sense, feel free to push a commit tweaking the message to what works for you, the virtiio-win-pkg-scripts mentions aren't necessary for my needs